Notes about DNS requirements for ACA clusters with vnets with custom DNS (#128272)
Azure/azure-content: articles/container-apps/private-endpoints-with-dns.md

> [!IMPORTANT]
> For organizations with strict DNS security requirements (such as banking and healthcare), Dedicated workload profiles provide the option to completely control DNS traffic flow through your custom DNS servers without requiring Azure Platform DNS access.

> [!IMPORTANT]

> > Users of private DNS zones MUST NOT block or override the resolution of `*.hcp.<LOCATION>.azmk8s.io`, `mcr.microsoft.com` and other DNS requirements that are shared with AKS and listed on [Azure Global required FQDN / application rules](../../aks/outbound-rules-control-egress#azure-global-required-fqdn--application-rules). Failure to ensure resolvability of required entries disrupts your Container Apps environment operation and networking.

Reviewer: ```suggestion
> Users of private DNS zones MUST NOT block or override the resolution of `*.hcp.<LOCATION>.azmk8s.io`, `mcr.microsoft.com` and other DNS requirements that are shared with AKS and listed on [Azure Global required FQDN / application rules](/azure/aks/outbound-rules-control-egress#azure-global-required-fqdn--application-rules). Failure to ensure resolvability of required entries disrupts your Container Apps environment operation and networking.
```

> > Users of private DNS zones MUST NOT block or override the resolution of `*.hcp.<location>.azmk8s.io`, `mcr.microsoft.com` and other DNS requirements that are shared with AKS and listed on [Azure Global required FQDN / application rules](../../aks/outbound-rules-control-egress#azure-global-required-fqdn--application-rules). Failure to ensure resolvability of those required entries will disrupt your Container App Environment operation and networking.

Author: I am not sure about the wording of "private DNS zones ". It is not about "private DNS zone" but overriding resolution for those domain names, right? How it is done (private DNS, forwarder configuration) isn't important, but that it is done somehow.

> > Users of private DNS zones MUST NOT block or override the resolution of `*.hcp.<location>.azmk8s.io`, `mcr.microsoft.com` and other DNS requirements that are shared with AKS and listed on [Azure Global required FQDN / application rules](../../aks/outbound-rules-control-egress#azure-global-required-fqdn--application-rules). Failure to ensure resolvability of those required entries will disrupt your Container App Environment operation and networking.

Reviewer 2: ```suggestion
> Users of private DNS zones MUST NOT block or override the resolution of `*.hcp.<LOCATION>.azmk8s.io`, `mcr.microsoft.com` and other DNS requirements that are shared with AKS and listed on [Azure Global required FQDN / application rules](../../aks/outbound-rules-control-egress#azure-global-required-fqdn--application-rules). Failure to ensure resolvability of required entries disrupts your Container Apps environment operation and networking.
```
